### ISSUE TITLE
It doesn't work

### ISSUE THREAD

**Issue by tyrion9** [AlexAkulov/putty-color-themes#4]
I run Windows 10 build April 2018,

I do exactly the same the instruction, but it doesn't work.

**Comment by ceritandogan** [comment]
@tyrion9  Which themes did you talk about and what is your putty version?
I have tried most of them. It works quite fine. 
I use  PuTTY Release 0.70.

**Comment by UNiXMIT** [comment]
I had issues with the instructions with regard to the .js file. My machine didn't know what to run this script with so instead I ran it in command prompt like this:

```
Cscript.exe "C:\Users\Admin\Downloads\putty-color-themes-master\_puttycolor.js" "C:\Users\Admin\Downloads\putty-color-t
hemes-master\44. Thayer.reg"
```

There might be an easier way than what I did but it worked for me.

**Comment by AlexAkulov** [comment]
Yes, `_puttycolor.js` is written on the JScript language embedded in Windows, it's not the JavaScript. 
By default js-files are associated with JScript in Windows but if you have ever installed any good text-editor you probably have js-files associated with JavaScript so `_puttycolor.js` does not work.
I will add to instruction an alternative way of run.

**Comment by abelardolg** [comment]
Hi there,
I followed the instructions but it didn't work.
![image](https://user-images.githubusercontent.com/22522883/48621387-b034ee00-e9a3-11e8-98c4-a1f2a35a405f.png)

**Comment by ysichrisdag** [comment]
To make it easier on yourself just open up a command prompt and navigate to the folder containing the .js and .reg file, then run the command with local paths:
`cscript _puttycolor.js "36. Seafoam Pastel.reg"`

**Comment by RoninUTA** [comment]
> To make it easier on yourself just open up a command prompt and navigate to the folder containing the .js and .reg file, then run the command with local paths: `cscript _puttycolor.js "36. Seafoam Pastel.reg"`

This method is the easiest. Still works as of 2023.
